addons/payment_custom/models/payment_provider.py

```diff
@@ -63,6 +64,17 @@ def action_recompute_pending_msg(self):
                     f'<p>{_("Please use the order name as communication reference.")}</p>' \
                     f'</div>'
 
+    @api.model
+    def _get_removal_domain(self, provider_code):
+        return OR([
```

Comment:
I find this `OR` a bit odd. Wouldn't it be cleaner and more robust to pass `provider_code='custom', custome_mode='sepa'` to `_remove_provider` so that we can directly return the right domain if `custom_mode` is set, and `super()` if not?

Reply:
Then we'd have a mention of `custom_mode` inside payment :thinking:, and also not sure modifying the API of `_remove_provider` in stable is a good idea (ok, parameter with a default value, but still).

So whatever, I don't really care, but I think this approach is the minimalist, without really risking much imho in term of robustness

Reply:
In `payment` it would be `**kwargs`. Maybe you can go with the `OR` in stable and switch to `kwargs` in master.